### ISSUE TITLE
CLI MindMap export functionality

### ISSUE THREAD

**Issue by naquad** [phase1geo/Minder#486]
Hi,

The app looks awesome and covers almost all the needs a casual user may have. But there's one thing missing:

For sake of integration with other tools (such as desktop Wiki or other text editing apps), a CLI interface for exporting MindMap in a specified format is needed,

An example of usage could be:

```
$ com.github.phase1geo.minder --export --format png [extra params for the format] /path/to/file.minder /path/to/file.png 
```

Please consider adding such functionality.

Thank you.

**Comment by phase1geo** [comment]
The zoom shortcuts are already available in the General pane.  I'll make a few changes for the other requests right now.

**Comment by phase1geo** [comment]
I have added the feature to the master branch.

**Comment by distbit0** [comment]
Also kind of related to this is the ability to import markdown files and convert them to mindmaps. 

I actually wrote a python script which does this a while back (sorry not on GH but can post if you'd like) and although it still works at the moment, the script I created for converting from mindmaps to markdown files has been broken by a change to the xml format.

As a result I am worried that my a similar fate awaits my markdown to mindmap conversion script and hence would love to see this functionality integrated natively via the cli into minder so I do not need to reverse engineer the xml format minder uses and maintain compatibility with the latest format.

Thanks so much for creating Minder btw. It is seriously the best mindmap software on linux and probably all platforms imo and is so great to use. 

Distbit

**Comment by distbit0** [comment]
My usecase is that I have dozens of documents I use to write my thoughts/take notes on certain subjects and sometimes prefer viewing/editing them in minder and sometime just as markdown. 

As a result, I have built a cron'd python script which syncs new changes to a minder file to a markdown file and vice versa. I can post this script if anyone is interested.

Currently it carries out both the converting of xml to markdown and markdown to xml, but it would be much easier if I did not need to do this and maintain compatibility, but instead could outsource this to the minder cli and just handle the syncing mechanism :)

Many thx
